csharp/ql/test/library-tests/dataflow/ssa/Capture.cs

```diff
@@ -236,4 +236,22 @@ void M1()
         M3();
         System.Console.WriteLine(i);
     }
+
+    void M2()
+    {
+        int i = 0;
+        void CaptureWrite()
+        {
+            i = 1;
+        }
+
+        void CaptureAndRef(ref int i)
+        {
+            CaptureWrite();
+            i = 2;
+        }
```

Comment:
The parameter in this local function could be named `j` to differentiate the `i = ...` assignments in `CaptureAndRef` and `CaptureWrite`.

Reply:
Good idea.